server/package.json

```diff
@@ -30,10 +30,12 @@
     "apollo-server-express": "^3.4.0",
     "axios": "^0.21.1",
     "connect-pg-simple": "^6.2.1",
+    "cross-spawn": "^7.0.3",
     "dotenv": "^8.2.0",
     "dotenv-cli": "^4.0.0",
     "express": "^5.0.0-alpha.8",
     "express-session": "^1.17.2",
+    "fs-extra": "^10.0.1",
```

Comment:
Noticed this dependency was missing despite being imported in `server/scripts/import-tests/index.js` so I've now added it.

I suppose it was being used through another package's dependencies?